ruby/vampires.rb

```diff
@@ -1,82 +1,115 @@
 # Create a program that asks a series of questions,
 	# Assign the input to a variable
+puts "How many employees would you like to process today?"
+employee_count = gets.to_i
 
-print "What is your name?"
-employee_name = gets.chomp
-case employee_name
-when "Drake Cula"
-	vamp_handle = true
-when "Tu Fang"
-	vamp_handle = true
-else 
-	vamp_handle = false
-end
+index = 0
 
-print "How old are you?" 
-age = gets.to_i
+while index < employee_count
 
-print "What year were you born?"
-birth_year = gets.to_i
-current_year = 2016
+	puts "What is your name?"
+	employee_name = gets.chomp
+	case employee_name
+	when "Drake Cula"

```

Comment:
I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.
```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```